generators/ruby/sdk/src/CustomConfig.ts

```diff
@@ -1,15 +1,17 @@
-import { BaseGeneratorConfigSchema } from "@fern-api/generator-commons";
+import { BaseGeneratorConfigSchema, ExtraDependenciesSchema } from "@fern-api/generator-commons";
 import { z } from "zod";
 
 export type RubySdkCustomConfig = z.infer<typeof RubySdkCustomConfigSchema>;
 export const RubySdkCustomConfigSchema = BaseGeneratorConfigSchema.extend({
     defaultTimeoutInSeconds: z.optional(z.union([z.literal("infinity"), z.number()])),
+    extraDevDependencies: z.optional(ExtraDependenciesSchema),
```

Comment:
should we do `extraDependencies` as well while we are here?

Reply:
It's in the base object, so it's already there!